spectrocloud/resource_cluster_nested.go

```diff
@@ -14,11 +14,11 @@ import (
 	"github.com/spectrocloud/terraform-provider-spectrocloud/pkg/client"
 )
 
-func resourceClusterNested() *schema.Resource {
+func resourceClusterVirtual() *schema.Resource {
 	return &schema.Resource{
-		CreateContext: resourceClusterNestedCreate,
-		ReadContext:   resourceClusterNestedRead,
-		UpdateContext: resourceClusterNestedUpdate,
+		CreateContext: resourceClusterVirtualCreate,
+		ReadContext:   resourceClusterVirtualRead,
+		UpdateContext: resourceClusterVirtualUpdate,
```

Comment:
```suggestion
		UpdateContext: resourceClusterVirtualUpdate,
		Description: "A resource to manage a Palette Virtual Cluster.",
```

@nikchern  I don't believe we have a template file for this resource. The same goes for `resource_application`

Reply:
correct we don't have. I will be adding them here as a part of the same PR.